src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -162,28 +174,41 @@ class ActivityFeed extends React.Component<Props, State> {
         this.setState({ mentionSelectorContacts: getMentionWithQuery(searchStr) });
     };
 
+    componentDidMount(): void {
+        this.attemptFeedSort(this.props);
+    }
+
+    componentWillReceiveProps(nextProps: any): void {
+        this.attemptFeedSort(nextProps);
+    }
+
+    attemptFeedSort(props: any): void {
```

Comment:
maybe call this shouldSortFeed, and return a boolean? The sort itself will always succeed, so attempt feels like weird wording.